msg/sensor_accel.msg

```diff
@@ -8,6 +8,9 @@ float32 y                 # acceleration in the FRD board frame Y-axis in m/s^2
 float32 z                 # acceleration in the FRD board frame Z-axis in m/s^2
 
 float32 temperature       # temperature in degrees Celsius
+float32 scale             # sensor resolution scale (LSB/g)
```

Comment:
Why include scale? I also don't see it being used below.

Reply:
Great point. This is a normal part of any sensor.  It seems appropriate to include.

Reply:
I think standardizing the non-fifo messages with the fifo counterparts merits including scale in the sensor_accel and sensor_gyro messages.  Let me know if you still think it's a bad idea.